{{cookiecutter.project_slug}}/README.md

```diff
@@ -130,6 +130,37 @@ You can automatically source from the `.envrc` file using [direnv](https://diren
 
 You can also manually export the variables to your environment using `source .envrc`
 
+
+
+### Debugging
+Debugging in development can be done using the following steps:
+
+#### Debugging Setup
+Update `k8s/base/app.configmap.yaml` with `data` field `DEBUGGER_IDE: "pycharm"`
+
+In PyCharm:
```

Comment:
Let's include a screenshot for more context. This will help one find similar UI if it ever changes.